packages/tailwindcss-x/.eslintrc.js

```diff
@@ -2,5 +2,14 @@ module.exports = {
   parserOptions: {
     tsconfigRootDir: __dirname,
     project: 'tsconfig.eslint.json'
-  }
+  },
+  overrides: [
+    {
+      files: ['src/**/*.ts'],
+      rules: {
+        '@typescript-eslint/no-unsafe-call': 'off',
+        '@typescript-eslint/no-unused-vars': 'off'
```

Comment:
I remember putting this rule at the beginning, but it seems it is no longer necessary and can be safely removed now.